docs/SUMMARY.md

```diff
@@ -39,7 +39,16 @@
     * [Excluding patterns](earthfile/earthignore.md)
 * [The `earthly` command](earthly-command/earthly-command.md)
 * [Configuration reference](earthly-config/earthly-config.md)
-* [CI integration](./ci-integration.md)
+* [CI integration](ci-integration/ci-integration.md)
+  * [Building An Image](ci-integration/building-an-image.md)
+  * [Pull-Through Cache](ci-integration/pull-through-cache.md)
+  * [Remote Buildkit](ci-integration/remote-buildkit.md)
+  * Guides
+    * [Jenkins](ci-integration/guides/jenkins.md)
+    * [Circle CI](ci-integration/guides/circle-integration.md)
+    * [GitHub Actions](ci-integration/guides/gh-actions-integration.md)
+    * [CodeBuild](ci-integration/guides/codebuild-integration.md)
```

Comment:
Given that some pages have moved, can you also update the redirects in `gitbook.yaml`?

Reply:
What redirects? Ill look around.